### PR TITLE
added macOS instructions

### PR DESCRIPTION
**Pull request by macmanx2**
Added macOS instructions to partially address #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,12 +26,31 @@ All profiles use DNS over HTTPS and support both IPv4 and IPv6.
 8. You'll see a warning that the profile might monitor your network traffic, and that it's unsigned. Tap **Install**.
 9. The profile will be installed. Tap **Done**.
 
+## Install on macOS 11 or higher
+
+1. Using Safari, choose the desired profile above.
+2. Right-click the **Raw** button and choose *Download Linked File*.
+3. Locate the downloaded file and remove the *.txt* file extension (so it ends in *.mobileconfig* only).
```
Comment (aheckler) [inline README.md]: Nitpicky, but can you use a code block `like this` instead of italics for *.txt* and *.mobileconfig*?

File: README.md [hunk 2]
```diff
@@ -26,12 +26,31 @@ All profiles use DNS over HTTPS and support both IPv4 and IPv6.
...
+4. Double-click the file, you'll see an alert telling you what to do next. Close that.
+5. Open the *Settings* app. There will be a new *Profiles* section. Open that.
```
Comment (aheckler) [inline README.md]: I'd use *System Preferences* here instead of *Settings*.

File: README.md [hunk 3]
```diff
@@ -26,12 +26,31 @@ All profiles use DNS over HTTPS and support both IPv4 and IPv6.
...
 
 1. Open *Settings* > *General* > *Profile*.
 2. Tap the profile.
 3. Tap **Remove Profile**.
 4. Enter your passcode.
 5. Tap **Remove**.
+
+## Uninstall on macOS 11 or higher
+
+1. Open *Settings* > *Profiles*.
```
Comment (aheckler) [inline README.md]: System Preferences here.

File: README.md (hunk 1 above)
Comment (macmanx2) [reply README.md]: addresses below, please use "correctly resolve feedback, lol" lol

**Comment by macmanx2** [comment]
addressed feedback, please use "correctly resolve feedback, lol" lol

**Pull request merged**